src/PlusDataCollection/vtkPlusGenericSerialDevice.cxx

```diff
@@ -230,6 +230,22 @@ PlusStatus vtkPlusGenericSerialDevice::SetRTS(bool onOff)
   return retval;
 }
 
+PlusStatus vtkPlusGenericSerialDevice::GetDSR(bool & onOff)
+{
+  // Either update or send commands - but not simultaneously
+  PlusLockGuard<vtkPlusRecursiveCriticalSection> updateMutexGuardedLock(this->Mutex);
+  PlusStatus retval = this->Serial->GetDSR(onOff);
+  return retval;
+}
+
+PlusStatus vtkPlusGenericSerialDevice::GetCTS(bool & onOff)

```

Comment:
Please include the 
//-------------------------------------------------------------------------

as per the convention